db/migrations/20220109181255_tickets.sql

```diff
@@ -0,0 +1,23 @@
+-- +goose Up
+CREATE TABLE IF NOT EXISTS public.tickets
+(
+    ticket_id integer NOT NULL GENERATED ALWAYS AS IDENTITY ( INCREMENT 1 START 1 MINVALUE 1 MAXVALUE 2147483647 CACHE 1 ),
+    "time" time with time zone NOT NULL,
```

Comment:
What is time column?
In general is better to avoid such names like `time`, `table`, `join` - they are used by postgres